src/payment/klarna-customer-experience.md

```diff
@@ -1,41 +1,40 @@
 ---
-title: Klarna Customer Experience
+title: Klarna Storefront Experience
 ---
 
-
-Klarna appears as a payment method during checkout, if enabled in the configuration. If data sharing is used, Klarna can instantly prequalify the customer, and present the available solutions.
+Klarna appears as a payment method during checkout, if it is enabled in the configuration. When data sharing is used, Klarna can instantly prequalify the customer and present the available solutions.
 
 ![Pay over time with Klarna]({% link images/images/storefront-cart-payment-method-klarna2.png %}){: .zoom}
-_Pay over time with Klarna_
+_Storefront - Pay over time with Klarna_
 
-## To pay with Klarna:
+## Pay with Klarna
 
-The payment solution presented might be one of the following:
+The payment solution presented to the customer might be one of the following:
 
 - Pay over time
 - Pay later
 
 ### Pay over time
 
-1. Select **Slice it. Pay over time with Klarna**.
+1. The customer selects **Slice it. Pay over time with Klarna**.
 
    ![Pay Over Time with Klarna]({% link images/images/storefront-cart-payment-method-klarna1.png %}){: .zoom}
    _Pay Over Time with Klarna_
 
-1. Do one of the following:
+1. The customer does one of the following:
 
-   - Add the purchase to your Klarna account.
-   - Apply for Klarna credit, and receive an instant decision.
+   - Adds the purchase to their Klarna account
+   - Applies for Klarna credit, and receives an instant decision
 
-1. After the terms of the credit are approved, tap <span class="btn">Place Order</span>.
+1. After the terms of the credit are approved, the customer clicks/taps <span class="btn">Place Order</span>.
```

Comment:
are we switching from click to "click/tap"?   first time i've seen this.

Reply:
Only when describing storefront functionality, which is more likely to be on a mobile device.